src/BetterPhpDocParser/PhpDocParser/StaticDoctrineAnnotationParser.php

```diff
@@ -21,6 +21,12 @@
  */
 final readonly class StaticDoctrineAnnotationParser
 {
+    /**
+     * @var string
+     * @see https://regex101.com/r/Pthg5d/1
+     */
+    private const END_OF_VALUE_CHARACTERS_REGEX = '/^[)} \n"\']+$/i';
```

Comment:
This is used to go past the end of the values for the annotation as the token iterator is not advanced to the end of these values

Reply:
windows new line need to be supported as well

```
'/^[)} \r?\n"\']+$/i';

Reply:
We don't need the ? as this just checks for any of these characters, so adding \r is enough, updated